### PR TITLE
Return events/alerts from signalflow events/alerts queries

### PR DESCRIPTION
**Pull request by scottlepp**
Support running signal flow queries like:
```
D = alerts(detector_name='Deployment').publish(label='D')
F = events(eventType='deployment').publish(label='F')
```

### CODE REVIEW

File: signalflow/computation.go [hunk 1]
```diff
@@ -352,6 +355,11 @@ func (c *Computation) Data() <-chan *messages.DataMessage {
 	return c.dataCh
 }
 
+// Events returns the results from events or alerts queries
+func (c *Computation) Events() []*messages.EventMessage {
+	return c.events
```
Comment (keitwb) [inline signalflow/computation.go:360]: This need to be synchronized and waited on like all the other methods that return execution metadata, such as https://github.com/signalfx/signalfx-go/blob/master/signalflow/computation.go#L117.

**Review by keitwb — CHANGES_REQUESTED** [review]
See comment.

**Comment by scottlepp** [comment]
@keitwb updated

**Review by keitwb — APPROVED** [review]
Looks good, thanks for  adding  this.

**Pull request merged**